receiver/hostmetricsreceiver/internal/scraper/processscraper/process_scraper_others.go

```diff
@@ -18,20 +18,51 @@
 package processscraper // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostmetricsreceiver/internal/scraper/processscraper"
 
 import (
```

Comment:
i would expect any code specific to `darwin` to go in `process_scraper_darwin.go` rather than other, then a `!darwin` added in line 15/16

Reply:
added and tested